config/gen-client.sh

```diff
@@ -15,7 +15,7 @@ openssl req -new -key ${USERNAME}.pem -out ${USERNAME}.csr -subj "/CN=${USERNAME
 
 # Generate a certificate for the client signed by the signer
```

Comment:
Let's change this comment to "Generate a certificate for the client signed by the root CA" to reflect the file name change?